### DISCUSSION TITLE
``` [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2873] [Discord Threads]
> 🙋‍♂️ **brunoqc**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Acedia**  
#### have you tried to manipulate root user from database directly?  
#  
> 🙋‍♂️ **brunoqc**  
#### No. I don't seem to have a root user. I checked `xc_users` and `nc_users_v2`.  
#  
> 🙋‍♂️ **Acedia**  
#### yes looks like the issue is caused because you don't have one, which database are you using as rootDB?  
#  
> 🙋‍♂️ **brunoqc**  
#### postgresql  
#  
> 🙋‍♂️ **Acedia**  
#### I will work on a fix for this case, but lets fix this for you for now  
#  
> 🙋‍♂️ **Acedia**  
#### do you have any spare user  
#  
> 🙋‍♂️ **Acedia**  
#### like any record on `nc_users_v2` will do it  
#  
> 🙋‍♂️ **brunoqc**  
#### yes I already have an account  
#  
> 🙋‍♂️ **Acedia**  
#### change that accounts roles column to `user,super`  
#  
> 🙋‍♂️ **Acedia**  
#### then restart your instance using env variables NC_ADMIN_EMAIL and NC_ADMIN_PASSWORD again  
#  
> 🙋‍♂️ **brunoqc**  
#### it worked. Thank you very much. Sorry for the delay. I had a call.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999381037358526485)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
